src/App.jsx

```diff
@@ -14,45 +16,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export function getReorderedGoods(
+  goods,
+  { sortType, isReversed },
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((f1, f2) => {
+    switch (sortType) {
+      case 'alphabet':
+        return f1.localeCompare(f2);
+      case 'length':
+        return f1.length - f2.length;
+      default:
+        return 0;
+    }
+  });
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
+
+  return visibleGoods;
+}
+
+export const App = () => {
+  const [sortType, setSortType] = useState('none');
```

Comment:
Consider using an empty string or null to explicitly show that there is no sorting applied